src/bin/pg_autoctl/cli_get_set_properties.c

```diff
@@ -133,9 +147,13 @@ CommandLine set_node_command =
 static CommandLine set_formation_number_sync_standby_command =
 	make_command("number-sync-standbys",
 				 "set number-sync-standbys for a formation on the monitor",
-				 CLI_PGDATA_USAGE "<number_sync_standbys>",
-				 CLI_PGDATA_OPTION,
-				 cli_getopt_pgdata,
+				 " [ --pgdata ] [ --json ] [ --formation ] [ --name ] "
```

Comment:
Does this one really need `--name`? Seems like it shouldn't, since `number-sync-standbys` is at the formation level.

Reply:
Fixed.